test/runtests.jl

```diff
@@ -1,6 +1,20 @@
+using DataFrames
+using Dates
+using DataStructures
+using Ipopt
 using MaximumGeneratorProfitTCRDD
+using PowerSystems
+using PowerSimulations
 using Test
+using TimeSeries
+
+#  Package Constants
+const PSY = PowerSystems
+const PSI = PowerSimulations
 
 @testset "MaximumGeneratorProfitTCRDD.jl" begin
-    # Write your tests here.
+    # Settings for testset
+    include("../examples/data/example_case4/buildup_case4.jl")
```

Comment:
I feel like we should be much more comprehensive here with testings. Especially if we're able to break down all the Power functions into separate pieces.